specification/graphrbac/data-plane/1.6/graphrbac.json

```diff
@@ -1757,6 +1885,27 @@
       },
       "description": "Active Directory error message."
     },
+    "DirectoryObject": {
+      "properties": {
+        "objectId": {
+          "type": "string",
+          "description": "The object ID.",
+          "readOnly": true
+        },
+        "objectType": {
+          "type": "string",
+          "description": "The object type.",
```

Comment:
`The object type. Possible values could be "User", "Application", "Group", "ServicePrincipal".`